README.md

```diff
@@ -21,34 +21,28 @@ Latest stable image: `registry.k8s.io/cloud-provider-gcp/gcp-compute-persistent-
-| HEAD Latest | HEAD (Migration ON) | [<img alt="Test Status" src="https://testgrid.k8s.io/q/summary/provider-gcp-compute-persistent-disk-csi-driver/Migration%20Kubernetes%20Master%20Driver%20Latest/tests_status" />](https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#Migration%20Kubernetes%20Master%20Driver%20Latest) |
-| HEAD stable-master | HEAD (Migration ON) | [<img alt="Test Status" src="https://testgrid.k8s.io/q/summary/provider-gcp-compute-persistent-disk-csi-driver/Migration%20Kubernetes%20Master%20Driver%20Stable/tests_status" />](https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#Migration%20Kubernetes%20Master%20Driver%20Stable) |
+| HEAD stable-master | HEAD (Migration ON) | [<img alt="Test Status" src="https://testgrid.k8s.io/q/summary/provider-gcp-compute-persistent-disk-csi-driver/Kubernetes%20Master%20Driver%20Latest%20Release/tests_status" />](https://testgrid.k8s.io/provider-gcp-compute-persistent-disk-csi-driver#Kubernetes%20Master%20Driver%20Latest%20Release) |
 
 ### CSI Compatibility
 
 This plugin is compatible with CSI versions [v1.2.0](https://github.com/container-storage-interface/spec/blob/v1.2.0/spec.md), [v1.1.0](https://github.com/container-storage-interface/spec/blob/v1.1.0/spec.md), and [v1.0.0](https://github.com/container-storage-interface/spec/blob/v1.0.0/spec.md)
 
-### Kubernetes Compatibility
-The following table captures the compatibility matrix of the core persistent disk driver binary
-`gke.gcr.io/gcp-compute-persistent-disk-csi-driver`
+### Kubernetes Version Recommendations
 
-| GCE PD CSI Driver\Kubernetes Version | 1.17+ |
-|--------------------------------------|-------|
-| v0.7.x (beta)                        | yes   |
-| v1.0.x (ga)                          | yes   |
-| dev                                  | yes   |
+The latest stable of this driver is recommended for the latest stable Kubernetes
+version. For previous Kubernetes versions, we recommend the following driver
+versions.
 
-The manifest bundle which captures all the driver components (driver pod which includes the containers csi-provisioner, csi-resizer, csi-snapshotter, gce-pd-driver, csi-driver-registrar;
-csi driver object, rbacs, pod security policies etc) can be picked up from the master branch [overlays](deploy/kubernetes/overlays) directory. We structure the overlays directory, per minor version of kubernetes because not all driver components can be used with all kubernetes versions.
-
-Example:
-
-`stable-1-21` overlays bundle can be used to deploy all the components of the driver on kubernetes 1.21.
+| Kubernetes Version | PD CSI Driver Version |
+|--------------------|-----------------------|
+| HEAD               | v1.10.2               |
```

Comment:
Maybe instead of the exact driver patch version, we can just specify the minor version, ie `1.10.x`. One less thing to update whenever we do a patch release.

Reply:
Good idea, done